### ISSUE TITLE
Supplement the dataset

### ISSUE THREAD

**Issue by DiegoJota** [NVIDIA/NeMo#2539]
Hi,

After fine tuning, if I want to supplement the dataset with a little more audio, should I restart training or can I restore the epoch where I left off?

**Comment by titu1994** [comment]
You can fine-tune with low learning rates for a short duration, however it is best to complete one training run, save the .Nemo file and then restore and fine-tune it some more. Otherwise pytorch lightning may behave unexpectedly

**Comment by DiegoJota** [comment]
Thanks for the answer, taking advantage of this question, assuming I have a large volume of data, around 10,000 hours, doing this on a relatively weak video card would take a long time, more than months...
If I want, can I split my base into several parts and do a workout for each of them, always restoring the model from the last workout I did? Or is it better if I do the full base training?
I'm building a dataset in Portuguese and I'm facing this problem
